client/src/components/ranking/RankingList.js

```diff
@@ -0,0 +1,61 @@
+
+import PropTypes from 'prop-types';
+import { Droppable, Draggable } from '@hello-pangea/dnd';
+import RecipeCard from '../recipeInfo/RecipeCard';
+
+const RankingList = ({ name, recipes, isDragDisabled }) => {
+
+  const getListStyle = isDraggingOver => ({
+    background: isDraggingOver ? '#EF960A' : '',
+    transition: 'background 0.4s ease',
+    padding: '20px',
```

Comment:
Could this as well as the `paddingBottom` below be a bit smaller?

Reply:
Modified paddign to be 10px and paddingBottom to be 15px